readme.md

```diff
@@ -32,6 +32,20 @@ sortKeys({c: 0, a: 0, b: 0}, {
 //=> {c: 0, b: 0, a: 0}
 ```
 
+As of v5.x, the comparator function, if given, will be bound to the current object.  It means `this` is available to access the object's values, e.g.:
+
+```js
+sortkeys({{c: 0, a: 0, b: 1}}, {
+  // compare based on keys and values
+  compare: function(left, right) {
+      let lvalue = this[left];
+      let rvalue = this[right];
+      if (lvalue === rvalue) return left.localeCompare(right);
+      return lvalue < rvalue;
+    }
+});
+```
+This can be used, e.g., to group all functions or objects last.  Do not use arrow function here since it would clobber `this`.
```

Comment:
This should be documented in the API section and should not mention it's new behavior. That's for the release notes to mention.